public/theme/boltcms-2021/listing.twig

```diff
@@ -0,0 +1,32 @@
+{# @var version \App\VersionControl #}
+{% extends 'partials/_master.twig' %}
+
+{% block main_container %}
+
+    <div class="content-wrapper">
+        <div class="current-version">
+            <p>Latest version of Bolt CMS: {{ get_version() }} <a href="https://github.com/bolt/core/releases/tag/{{ get_version() }}">Download this
+                    version</a></p>
+        </div>
+        <div class="important-message">
+            <div class="header-block">
+                <h2>Bolt 4.1.13 released</h2>
+                <span transition-style="in:circle:top-left">#Announcement</span>
+            </div>
+            <p>Bolt 4.1.13 brings a slew of accessibility, usability, and DX improvements.</p>
+        </div>
+        {% setcontent record = 'news' %}
```

Comment:
Rather than fetching content ourselves, let's use the global `records` variable.